scripts/dockerbot/check_prs.py

```diff
@@ -277,9 +306,17 @@ def check_status(pr):
                 logger.info('Setting task {} for pr {} to {}'
                             .format(task['stream'], pr['number'], states[task['result']]))
                 set_status(commit, states[task['result']], task['stream'])
+            if task['result'] == 'failed':
+                failed_streams.append(task['stream'])
```

Comment:
I think you want this up on line 303. It should only execute if the status is different from last time, else you're going to get spammed....a lot. Also it should probably be duped into line 308, which happens in the case that this PR doesn't have a status yet.